examples/rust/Earthfile

```diff
@@ -0,0 +1,13 @@
+FROM rust:1.24.0
+
+WORKDIR /usr/src/app
```

Comment:
We should use `/example-rust` for simplicity and consistency with other examples.